action.yml

```diff
@@ -62,7 +34,32 @@ inputs:
 
 outputs:
   response:
+    value: Done
     description: 'The result of migration checks'
+
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: 'composite'
+  steps:
+    - name: DB migration with DLE
+      uses: agneum/migration-action/core@composite-action
+      with:
+        commands: ${{ inputs.commands }}
+        dbname: ${{ inputs.dbname }}
+        migration_envs: ${{ inputs.migration_envs }}
+        download_artifacts: ${{ inputs.download_artifacts }}
```

Comment:
@agneum Should we distinguish "safe" (PII-free) and "unsafe" artifacts, for security reasons?

Reply:
Is there any way to distinguish them?